src/lib/RequestParser.js

```diff
@@ -169,11 +169,15 @@ class RequestParser { // eslint-disable-line no-unused-vars
 
     /**
      * @param {any} message
-     * @returns {Uint8Array}
+     * @returns {string | Uint8Array}
```

Comment:
I see that it is displayed in hex in Uint8Array case and as string otherwise. Is that a good enough criteria for the determination of whether it is data or displayable text?